Templates/HtmlSpoiler.tt

```diff
@@ -35,9 +70,9 @@
 <h2>Dungeon Entrance Replacements</h2>
 <table border="1" class="item-replacements">
 	<tr>
-		<th>Entrance</th>
-	    <th></th>
-		<th>New Destination</th>
+		<th id="help" class="light-mode">Entrance</th>
+	    <th id="help" class="light-mode"></th>
+		<th id="help" class="light-mode">New Destination</th>
```

Comment:
See previous comment, these should not require their own `light-mode | dark-mode` class if it's already applied to the body.

Reply:
That's what I originally tried but it would not update highlight colors. It only worked when I added it to have it's own light and dark mode.

Reply:
I removed the classes, and the corresponding javascript - seemed to work for me.

Reply:
Then I don't think I follow what you mean.